study-builder/studies/rarex/consent-parental-assent.conf

```diff
@@ -497,6 +497,66 @@
           "blockType": "QUESTION",
           "shownExpr": null
         },
+        {
+          "question": {
+            include required("../snippets/text-question-signature-required.conf"),
+            "stableId": "CONSENT_ASSENT_CHILD_SIGNATURE",
```

Comment:
Why did we move this out of section 4?

Reply:
This is definitely a mistake, this case is similar to the former case. we have two places where child signature is needed. One for "generally not being able to consent" and second for "generally not being able to consent" & "is able to read assent"

Reply:
I will add this to the section 4, but with different stableId